src/megatron/bridge/recipes/deepseek/deepseek_v3.py

```diff
@@ -118,14 +109,17 @@ def model_config(
     cfg.apply_rope_fusion = True
 
     # Pipeline parallelism configs. We infer PP layout from the provided PP and VP size
+    if mtp_num_layers is None:
```

Comment:
should this read from `use_mtp` like L92? is the `use_mtp` flag still useful, or would it make more sense to rely entirely on `mtp_num_layers` being not None?